apps/drupal/webpack.drupal.dev.js

```diff
@@ -24,7 +24,7 @@ const dev = {
   plugins: [
     new WebpackShellPlugin({
       onBuildEnd: [
-        'echo \nWebpack drupal dev build complete! Edit apps/drupal/webpack.drupal.dev.js to replace this line with `drupal cr all` now.',
+        'drupal cr all',
```

Comment:
Let's change this so that we have the echo, but also the `drupal cr all` commented out. Make a note that a user will need to uncomment it to use it. (We can't assume everyone will have drupal console installed.

Reply:
I'm not sure what the trends are, but my expectation is still that Drush is more common.